src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -12,21 +12,28 @@
-    public static final Block blockHutMiner          = new BlockHutMiner();
-    public static final Block blockHutLumberjack     = new BlockHutLumberjack();
-    public static final Block blockHutBaker          = new BlockHutBaker();
-    public static final Block blockHutBuilder        = new BlockHutBuilder();
-    public static final Block blockHutWarehouse      = new BlockHutDeliveryman();
-    public static final Block blockHutBlacksmith     = new BlockHutBlacksmith();
-    public static final Block blockHutStonemason     = new BlockHutStonemason();
-    public static final Block blockHutFarmer         = new BlockHutFarmer();
-    public static final Block blockHutFisherman      = new BlockHutFisherman();
-    public static final Block blockSubstitution      = new BlockSubstitution();
-    public static final Block blockSolidSubstitution = new BlockSolidSubstitution();
-    public static final Block blockHutField          = new BlockHutField();
-    public static final Block blockHutGuardTower     = new BlockHutGuardTower();
+    public static final Block blockHutTownHall            = new BlockHutTownHall();
+    public static final Block blockHutCitizen             = new BlockHutCitizen();
+    public static final Block blockHutMiner               = new BlockHutMiner();
+    public static final Block blockHutLumberjack          = new BlockHutLumberjack();
+    public static final Block blockHutBaker               = new BlockHutBaker();
+    public static final Block blockHutBuilder             = new BlockHutBuilder();
+    public static final Block blockHutDeliveryman         = new BlockHutDeliveryman();
+    public static final Block blockHutBlacksmith          = new BlockHutBlacksmith();
+    public static final Block blockHutStonemason          = new BlockHutStonemason();
+    public static final Block blockHutFarmer              = new BlockHutFarmer();
+    public static final Block blockHutFisherman           = new BlockHutFisherman();
+    public static final Block blockSubstitution           = new BlockSubstitution();
+    public static final Block blockSolidSubstitution      = new BlockSolidSubstitution();
+    public static final Block blockHutField               = new BlockHutField();
+    public static final Block blockHutGuardTower          = new BlockHutGuardTower();
+    public static final Block blockHutWareHouse           = new BlockHutWareHouse();
+    public static final Block blockConstructionTape       = new BlockConstructionTape();
+    public static final Block blockConstructionTapeCorner = new BlockConstructionTapeCorner();
+    public static final Block blockTimberFrame            = new BlockTimberFrame();
+    public static final Block blockRack                   = new BlockMinecoloniesRack();
+    public static final Block blockWayPoint               = new BlockWaypoint();
+
 
     // Deactivated for now
     // public static final Block blockBarrel        = new BlockBarrel();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)